ros_msft_onnx/src/ros_msft_onnx.cpp

```diff
@@ -298,7 +298,7 @@ bool OnnxTracker::init(ros::NodeHandle& nh, ros::NodeHandle& nhPrivate)
     std::string trackerType;
     if (nhPrivate.getParam("tracker_type", trackerType))
     {
-        if (trackerType == "yolo")
+        if (trackerType == "yolo" || trackerType == "customvision")
```

Comment:
tracker Type is intended to specify a type of pre/post processing. if they are the same, we shouldn't use a new type.